pkg/cloudprovider/aws/instance.go

```diff
@@ -17,11 +17,10 @@ package aws
 import (
 	"context"
 	"fmt"
-	"math/rand"
-	"strings"
-
 	"go.uber.org/zap"
 	v1 "k8s.io/api/core/v1"
+	"math/rand"
+	"strings"
```

Comment:
I think we should look into whats causing this, because these are std library packages and the convention is -
```
std library imports

project-specific imports

third-party imports
```
Refer- https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Looks like `make ci` doesn't sort the imports by separating it like that. It only sorts the imports alphabetically unless there's a newline in between. Maybe this is auto done for VSCode, but not for Goland.